docs/animations/ImplicitAnimations.md

```diff
@@ -7,6 +7,9 @@ keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp
 
 # Implicit Composition Animations in XAML
 
+> [!WARNING]

```

Comment:
The examples below are still workable in the new world, eh? But not in the new `ImplicitAnimationSet` document? Do we really need a new doc or can we just re-use the old one here and update it? (Just want to try and maybe create a bit less churn if we can?)

Reply:
They should roughly work without XAML changes I think (with some minor differences in some cases).
I just split the documents to clearly separate the various types though?
As in, now we have the implicit/explicit sets, whereas before there was just that single set, and the various docs pages had more generic titles and names (eg. "composition animations"). My rationale here was that just using the type name for the file name and title would've made things easier to follow? Also to be consistent with the other docs from other sections 🤔
Thoughts?